_posts/2021-03-09-stellar-issue-asset.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: How to issue an asset on Stellar
+subtitle: AFFEN goes live
+tags: [poc, blockchain, stellar, token, asset, github]
+author: gmzabos
+---
+
+## Intro - what is Stellar?
+
+[Stellar](http://stellar.org/){:target="_blank"} is an open-source network for currencies and payments. It is public, there is no "real" owner as it is a decentralized network. See the [Intro](https://stellar.org/learn/intro-to-stellar){:target="_blank"} on their website, as it speaks for itself better than i could do.
+
+## Intro - why do we issue our own asset?
+
+We came up with the idea while experimenting with Stellar (and other blockchains) and decided to create an asset (or "token") to hook it up to [codeaffen](https://github.com/codeaffen){:target="_blank"} on [github](https://github.com/){:target="_blank"}. The token would function within that ecosystem as a reward for the sake of gamification while submitting code & contributing in other ways to the project. Every good role-playing game has its own XP, you get the idea...
+The token was named [AFFEN](https://stellar.expert/explorer/public/asset/AFFEN-GDIZF5MAUDQ6TLIK3PNZ2ESOJAZDNZWIB675S4SQ5RWUTZP3B3ONUPMX){:target="_blank"} (german: "monkey") and is represented by that neat and busy code monkey on our [logo](https://codeaffen.org/assets/img/Codeaffen_only_C_wo_bg.png){:target="_blank"}. There is a supply of 1,000,000 code monkeys to give away. AFFEN, anyone?
+
+## Intro - where to start?
+
+Best starting point is the official [documentation](https://developers.stellar.org/docs/issuing-assets/){:target="_blank"}, but there is also great public articles & videos on the topic. I recommend reading at least into the documentation, just to get a feeling what to do and why to do it. From there on...
+
+## Howto - let's go
+
+- You will need an active wallet, which you use to fund two new accounts with. Make sure you have 5 XLM in that wallet which you can transfer out. I used my [Lobstr](https://lobstr.co/){:target="_blank"} wallet for this.
```

Comment:
Maybe a small description what `XLM` is should be useful

Reply:
I'll skip this one - documentation provides it (see link in intro)